CHANGELOG.md

```diff
@@ -0,0 +1,82 @@
+# [1.3.0-beta.0](https://github.com/algolia/search-insights.js/compare/v1.2.0...v1.3.0-beta.0) (2019-09-09)
+
+
+### Bug Fixes
+
+* remove startWith and use indexOf instead ([#129](https://github.com/algolia/search-insights.js/issues/129)) ([9c79e20](https://github.com/algolia/search-insights.js/commit/9c79e20))
+* **bug:** persist function reference after library is loaded ([#128](https://github.com/algolia/search-insights.js/issues/128)) ([662bf64](https://github.com/algolia/search-insights.js/commit/662bf64))
+* **compat:** node compatibility ([#140](https://github.com/algolia/search-insights.js/issues/140)) ([e804bd1](https://github.com/algolia/search-insights.js/commit/e804bd1)), closes [#141](https://github.com/algolia/search-insights.js/issues/141) [#143](https://github.com/algolia/search-insights.js/issues/143) [#144](https://github.com/algolia/search-insights.js/issues/144) [#142](https://github.com/algolia/search-insights.js/issues/142)
+* **getUserToken:** add integration tests ([#114](https://github.com/algolia/search-insights.js/issues/114)) ([c02c178](https://github.com/algolia/search-insights.js/commit/c02c178))
+* **processQueue:** move callback on method level ([#115](https://github.com/algolia/search-insights.js/issues/115)) ([1ff8191](https://github.com/algolia/search-insights.js/commit/1ff8191))
+
+
+
+# [1.2.0](https://github.com/algolia/search-insights.js/compare/v1.1.1...v1.2.0) (2019-04-24)
+
+
+### Bug Fixes
+
+* **processQueue:** extract globalObject from processQueue for easier testing ([#106](https://github.com/algolia/search-insights.js/issues/106)) ([9b56fd9](https://github.com/algolia/search-insights.js/commit/9b56fd9))
```

Comment:
This one can be removed, it's an internal change, it should have been a chore but I guess I didn't tag it correctly: https://github.com/algolia/search-insights.js/pull/106